src/main/java/com/spartronics4915/frc2020/RobotContainer.java

```diff
@@ -49,6 +53,9 @@ public AutoMode(String name, Command command)
      */
     public RobotContainer()
     {
+        // TODO: verify subsystem instantiation standardization
```

Comment:
Also confusing. 'standardization'?

Reply:
I think TODO's should give enough information for uninformed/new students to act upon them, if possible. Most TODO's won't be addressed by this year's students, based on my experience. Someone later will pick this up, and may not get direction from their leadership.

Reply:
Again, meant for @j-james on how the subsystems are instantiated -- in the past we used singleton method